android/README.md

```diff
@@ -0,0 +1,456 @@
+# Developer Documentation - For Android SDK
+
+## Introduction
+
+The Asgardeo Auth Android SDK enables Android applications (written in Kotlin) to utilize OpenID Connect (OIDC) authentication with the WSO2 Identity Server/Asgardeo serving as the Consumer Identity and Access Management (CIAM) Provider through API-based authentication. This SDK assists you in creating custom login flows directly within the applications themselves, without relying on browser redirects, thereby prioritizing user experience.
+
+## Prerequisite
+
+- Supports Android applications written in Kotlin programming language.
+- The minimum supported SDK is API level 26, compiled to API level 34. However, there are certain limitations when using specific authentication methods:
+    - Passkeys are supported on API level 34 and above.
+    - Google authentication using the Credential Manager API is supported on API level 34 and above.
+
+## Getting Started
+
+### Register your application in /WSO2 Identity Server/Asgardeo
+
+#### WSO2 Identity Server
+
+1. Download the latest version of WSO2 Identity Server, and start the WSO2 Identity Server.
+2. Register a Mobile Application to integrate your application with WSO2 Identity Server. You will obtain a `client_ID` from WSO2 Identity Server for your application which will need to be embedded later for the SDK integration. Also note the redirect URI that you used to create the application, this is also required for the SDK integration.
+3. In the created mobile application go to the "Advanced" section and enable the application native authentication for your Android application.
+
+#### Asgardeo
+
+1. Register to Asgardeo and create an organization if you don't already have one. The organization name you choose will be referred to as `<org_name>` throughout this document.
+2. Register a Mobile Application in Asgardeo to integrate your application with Asgardeo. You will obtain a `client_ID` from Asgardeo for your application which will need to be embedded later for the SDK integration. Also note the redirect URI that you used to create the application, this is also required for the SDK integration.
```

Comment:
Maybe we can link to the relevant asgardeo docs as well?

Reply:
Fixed with https://github.com/asgardeo/mobile-ui-sdks/pull/1/commits/5a909401570da5c711812c97884fdb49afc7c335